include/sys/fs/zfs.h

```diff
@@ -741,6 +744,24 @@ typedef enum zpool_errata {
 	ZPOOL_ERRATA_ZOL_2094_ASYNC_DESTROY,
 } zpool_errata_t;
 
+#ifndef BIT /* Include BIT() for userspace */
+#define	BIT(n)	(1ULL << (n))
+#endif
+
+enum vdev_stat_flags {VS_DEFAULT = 0, VS_LATENCY = 1, VS_QUEUES = 2,
+		VS_HISTO = 3, VS_FLAGS_COUNT /* always last element */};
+
+/* Mask versions of all our flags */
+#define	VS_DEFAULT_M	BIT(VS_DEFAULT)
+#define	VS_LATENCY_M	BIT(VS_LATENCY)
+#define	VS_QUEUES_M	BIT(VS_QUEUES)
+#define	VS_HISTO_M	BIT(VS_HISTO)
```

Comment:
These names may be a bit to generic.  It's conceivable that we'll want to add additional statistics, for example a per-vdev histogram which measures something else we're interested in.